charmcraft/commands/store/__init__.py

```diff
@@ -827,6 +827,93 @@ def run(self, parsed_args):
                 lib_data.full_name, lib_data.api, lib_data.patch)
 
 
+class FetchLibCommand(BaseCommand):
+    """Fetch one or more charm libraries."""
+    name = 'fetch-lib'
+    help_msg = "Fetch one or more charm libraries"
+    overview = textwrap.dedent("""
+        Fetch charm libraries.
+
+        It will download the library the first time, and update it the next times.
```

Comment:
Questions/comments:

- maybe something like "subsequent fetches of the same libraries will update the local copies"
- the libraries are always downloaded, not just the first time, right?
- if wanting to keep the current message then s/times/time/

Reply:
Reading the code below, maybe clarify when local gets updated when fetching

Reply:
Rewrote it: *The first time a library is downloaded the command will create the needed directories to place it, subsequent fetches will just update the local copy.*